shell/browser/ui/file_dialog_gtk.cc

```diff
@@ -1,395 +0,0 @@
-// Copyright (c) 2014 GitHub, Inc.
-// Use of this source code is governed by the MIT license that can be
-// found in the LICENSE file.
-
-#include <memory>
-#include <string>
-
-#include "base/files/file_util.h"
-#include "base/functional/bind.h"
-#include "base/functional/callback.h"
-#include "base/memory/raw_ptr.h"
-#include "base/memory/raw_ptr_exclusion.h"
-#include "base/strings/string_util.h"
-#include "electron/electron_gtk_stubs.h"
```

Comment:
Can also remove `https://github.com/electron/electron/blob/main/shell/browser/ui/electron_gtk.sigs` as part of this change.